### ISSUE TITLE
[WinUI 3 Gallery: Design-> Spacing]: Alt text is not defined for the images present under "Page with cards layout" and "Form Layout" headings.

### ISSUE THREAD

**Issue by sabareesanms** [microsoft/WinUI-Gallery#1967]
**Environment Details:**

**OS Version:** Dev (OS Build 27881.1000)

**App:** WinUI 3 Gallery

**App Version:** 2.6.0.0 (SDK 1.7)

**AT:** Screen Reader

**Pre-Requisite:**

Turn on Narrator (Ctrl + Win + Enter).

**Repro Steps:**

**Actual Result:** UI 3 Gallery app. Navigate to search box using tab key. Type 'Spacing' in search box and hit enter. Navigate to 'Spacing' card and activate it. Now navigate to the images present under "Page with cards layout" and "Form Layout" headings using caps + arrow keys.  Actual Result:

**Actual Result:**

**Actual Result:**

**Actual Result:**

**Actual Result:**

**Expected Result:** defined for the images present under "Page with cards layout" and "Form Layout" headings. When user navigates to it using caps + arrow keys, narrator is announcing as "Image".  Expected Result: Alt text should be defined for the images present under "Page with cards layout" and "Form Layout" headings.  User Impact:

Alt text is not defined for the images present under "Page with cards layout" and "Form Layout" headings. When user navigates to it using caps + arrow keys, narrator is announcing as "Image".

**Expected Result:**

Alt text should be defined for the images present under "Page with cards layout" and "Form Layout" headings.

**User Impact:**

**User Impact:**

**MAS Reference:** ual impairments who rely on screen reader will be impacted if alt text is not defined to an image. User will not be able to the context of the image.   MAS Reference: MAS 1.1.1 – Non-text Content  Trap ID: 1.6 - Poor Grouping

**MAS Reference:** ual impairments who rely on screen reader will be impacted if alt text is not defined to an image. User will not be able to the context of the image.   MAS Reference: MAS 1.1.1 – Non-text Content  Trap ID: 1.6 - Poor Grouping

**MAS Reference:** ual impairments who rely on screen reader will be impacted if alt text is not defined to an image. User will not be able to the context of the image.   MAS Reference: MAS 1.1.1 – Non-text Content  Trap ID: 1.6 - Poor Grouping

**MAS Reference:** ual impairments who rely on screen reader will be impacted if alt text is not defined to an image. User will not be able to the context of the image.   MAS Reference: MAS 1.1.1 – Non-text Content  Trap ID: 1.6 - Poor Grouping

**MAS Reference:** ual impairments who rely on screen reader will be impacted if alt text is not defined to an image. User will not be able to the context of the image.   MAS Reference: MAS 1.1.1 – Non-text Content  Trap ID: 1.6 - Poor Grouping

Users with visual impairments who rely on screen reader will be impacted if alt text is not defined to an image. User will not be able to the context of the image.

**MAS Reference:**

MAS 1.1.1 – Non-text Content

**Trap ID:**

1.6 - Poor Grouping

**“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”**
